patches/minecraft/net/minecraft/block/BubbleColumnBlock.java.patch

```diff
@@ -0,0 +1,20 @@
+--- a/net/minecraft/block/BubbleColumnBlock.java
++++ b/net/minecraft/block/BubbleColumnBlock.java
+@@ -81,7 +81,7 @@
+       if (block == Blocks.field_203203_C) {
+          return blockstate.func_177229_b(field_203160_a);
+       } else {
+-         return block != Blocks.field_150425_aM;
++         return block.getDrag();
+       }
+    }
+ 
+@@ -126,7 +126,7 @@
+ 
+    public boolean func_196260_a(BlockState p_196260_1_, IWorldReader p_196260_2_, BlockPos p_196260_3_) {
+       Block block = p_196260_2_.func_180495_p(p_196260_3_.func_177977_b()).func_177230_c();
+-      return block == Blocks.field_203203_C || block == Blocks.field_196814_hQ || block == Blocks.field_150425_aM;
++      return block == Blocks.field_203203_C || block.isBubbleSource();
```

Comment:
This doesn't support custom bubble column blocks, as in blocks that act as a bubble relay, but not a source.